### ISSUE TITLE
Set up automated regression tests

### ISSUE THREAD

**Issue by mariagould** [CDLUC3/ezid#322]
Based on tests documented in #140, set up automated tests to perform after patching

Possible tests:

- [x] EZID app status on the server
- [ ] EZID app logs
- [x] Status of background jobs
- [x] (prod): LC status
- [x] UI > search: random keyword
- [x] mint - sample DOI and ARK on test shoulder

**Comment by jsjiang** [comment]
Script:
* https://github.com/CDLUC3/ezid-ops/blob/main/scripts/verify_ezid_after_patching.py

Output on ezid-dev:
```
ok 1 - Verify EZID status
ok 2 - Verify search function
## Create identifier
ok 3.1 - doi:10.15697/FK2C31K created
ok 3.2 - doi:10.15697/FK27880 created
ok 3.3 - ark:/99999/fk4572tn0v created
ok 3.4 - doi:10.5072/FK26W9FZ0F created
ok 3.5 - doi:10.5072/FK2377FH9R created
ok 3.6 - ark:/99999/fk41g23t5q created
ok 3.7 - doi:10.5072/FK2ZG6QW4X created
ok 3.8 - ark:/99999/fk4wt03k4d created
## On dev: Skip Check background job status
```

Output on ezid-stg:
```
ok 1 - Verify EZID status
ok 2 - Verify search function
## Create identifier
ok 3.1 - doi:10.15697/FK2D63W created
ok 3.2 - doi:10.15697/FK28H13 created
ok 3.3 - ark:/99999/fk4jt16919 created
ok 3.4 - doi:10.5072/FK2VH5P857 created
ok 3.5 - doi:10.5072/FK2QR4ZG00 created
ok 3.6 - ark:/99999/fk4f20gg7m created
ok 3.7 - doi:10.5072/FK2M047N6Q created
ok 3.8 - ark:/99999/fk4989rp5p created
## Check background job status
ok 4.1 - ezid-proc-binder active running
ok 4.2 - ezid-proc-crossref active running
ok 4.3 - ezid-proc-datacite active running
ok 4.4 - ezid-proc-download active running
ok 4.5 - ezid-proc-expunge active running
ok 4.6 - ezid-proc-newsfeed active running
ok 4.7 - ezid-proc-search-indexer active running
ok 4.8 - ezid-proc-stats active running
info 4.9 - ezid-proc-link-checker is not running
info 4.10 - ezid-proc-link-checker-update is not running
```

**Comment by jsjiang** [comment]
* Created a new public repo "ezid-ops-scripts" to host EZID operational scripts that can be shared with the public. 
* The repo is deployed by puppet and are on all uc3-ezid-ui hosts (ezid-dev/stg/prd)

**Comment by jsjiang** [comment]
To do:
* Add output EZID version to the script
* Deploy the change through puppet

**Comment by jsjiang** [comment]
Add following items:
* show EZID version
* update DOI  and/or ARK

**Comment by jsjiang** [comment]
released as v0.0.2
